src/components/DpDataTableExtended/DpDataTableExtended.vue

```diff
@@ -35,18 +35,10 @@
     </dp-sticky-element>
 
     <dp-data-table
-      :has-flyout="hasFlyout"
-      :header-fields="headerFields"
-      :is-expandable="isExpandable"
-      :is-loading="isLoading"
-      :is-resizable="isResizable"
-      :is-selectable="isSelectable"
-      :is-truncatable="isTruncatable"
```

Comment:
Just to simplify further adjustments. Not sure if this is a clever move

Reply:
i think it is ok. is there a reason why you define `should-be-selected-items` on its own?

Reply:
It was named differently before so I can't just pass it along. And I don't want to refactor it with this change.